inductiva/molecules/scenarios/protein_solvation/protein_solvation.py

```diff
@@ -43,16 +43,16 @@ def __init__(
     def simulate(
             self,
             simulator: Simulator = GROMACS(),
-            working_dir: Optional[Path] = None,
+            output_dir: Optional[Path] = None,
             simulation_time: float = 10,  # ns
             integrator: str = "md",
             nsteps_minim: int = 5000):
         """Simulate the solvation of a protein.
         Args:
-            working_dir: The working directory where the simulation 
-            will be executed.
+            output_dir: The  output directory for the simulation.
             simulation_time: The simulation time in ns. Default is 10 ns.
```

Comment:
No need to state the default value. That's given in the argument list. The same applies to other arguments.